resteasy-jaxrs/src/main/java/org/jboss/resteasy/spi/Registry.java

```diff
@@ -105,4 +105,6 @@
    void addJndiResource(String jndiName, ResourceClass resourceClass);
 
    void addJndiResource(String jndiName, ResourceClass resourceClass, String basePath);
+
+   void checkAmbiguousUri();

```

Comment:
@asoldano @rsearls Whouldn't be better to use default interface method? It would be backward compatible, Registry is public interface. This default method could be just empty, like VertxRegistry.java has.

This commit is present in 3.6 and master ...